src/Chunk.cpp

```diff
@@ -640,6 +640,7 @@ void cChunk::Tick(std::chrono::milliseconds a_Dt)
 		else if ((*itr)->IsWorldTravellingFrom(m_World))
 		{
 			// Remove all entities that are travelling to another world
+			LOGD("Removing entity from %d,%d that's travelling between worlds.", m_PosX, m_PosZ);
```

Comment:
Usually we log chunk coords enclosed in `[]` and block coords enclosed in `{}`. Also, add a space after the comma.